app/src/process/ActivateAccountProcess.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import * as AccountAPI from '../api/AccountAPI'
+import { Redirect, Route } from 'react-router-dom'
+import WelcomePage from '../containers/WelcomePage'
+
+
+class ActivateAccountProcess extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      isFetching: true,
+      success: false
+    };
+  }
+
+  componentDidMount(){
+    const {onSetupUserInfo, onSetupApiToken} = this.props
+    const {token} = this.props.computedMatch.params
+
+    AccountAPI.activateAccount(token)
+    .then(res => {
+      onSetupApiToken(token)
+      onSetupUserInfo(res.data)
+      // TODO: change back to true after testing
```

Comment:
Same, test code should be on you local only.

Reply:
fix